### ISSUE TITLE
voc dataset problem

### ISSUE THREAD

**Issue by Radleye** [ultralytics/yolov5#2388]
## ❔Question
As the paper  mentioned
 'The deep neural networks were trained using the VOC2007/2012 training datasets, and the
mean average precision (mAP) was computed on the VOC2007 test dataset to evaluate the object
detection accuracy of the deep neural networks, as is standard practice in research literature.
how i should set it in yolov5'
I want to know if the default config in voc.yaml is same as above set. beacuse i find the default set is to use voc2007 test dataset to val rather than test.and how i can test the mode performance because the all the dataset is to use train and val . i just want toknow what's the standard practice in research literature?16651 pictures is the voc07+12 trainval dataset to train ,and 4925 pictures is the voc07 test dataset to val  in default voc.yaml in yolov5 . who know this, thanks.

**Comment by glenn-jocher** [comment]
@xiaowo1996 to train on VOC run this command. Dataset autodownloads using https://github.com/ultralytics/yolov5/blob/master/data/scripts/get_voc.sh

```
python train.py --data voc.yaml
```

**Comment by Radleye** [comment]
@glenn-jocher  yes, I have already autodownload the voc dataset and train sucessfully same as you do,but the question is whether this is standard practice as paper mentioned:'The deep neural networks were trained using the VOC2007/2012 training datasets, and themean average precision (mAP) was computed on the VOC2007 test dataset to evaluate the object detection accuracy of the deep neural networks, as is **_standard practice in research literature_**'.beacuse i find there is no test dataset in default voc.yaml

**Comment by glenn-jocher** [comment]
@xiaowo1996 what paper?

**Comment by Radleye** [comment]
Yolo nano,I want use edited yolov5 do a experiment compare with it. but i don't understand its experiment detail about voc dataset,what is the standard practice in research literature as the paper mentioned? .same as yolov5 default voc.yaml?. I capture the experiment screen
![2](https://user-images.githubusercontent.com/24860639/110264864-39e5ab00-7ff5-11eb-9ea7-9a87fef691db.png)

**Comment by glenn-jocher** [comment]
@xiaowo1996 sorry, I'm not sure, I'm not familiar with the paper. You can look through the file that generates the VOC dataset here: https://github.com/ultralytics/yolov5/blob/master/data/scripts/get_voc.sh

https://github.com/ultralytics/yolov5/blob/ba18528b4737a4b08b55653c54f3d3e830f8e151/data/scripts/get_voc.sh#L1-L139

**Comment by Radleye** [comment]
ok ,fine




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Glenn Jocher"<notifications@github.com&gt;; 
发送时间: 2021年3月8日(星期一) 上午10:08
收件人: "ultralytics/yolov5"<yolov5@noreply.github.com&gt;; 
抄送: "菊次郎的夏天"<429740343@qq.com&gt;; "Mention"<mention@noreply.github.com&gt;; 
主题: Re: [ultralytics/yolov5] voc dataset problem (#2388)





 
@xiaowo1996 sorry, I'm not sure, I'm not familiar with the paper. You can look through the file that generates the VOC dataset here: https://github.com/ultralytics/yolov5/blob/master/data/scripts/get_voc.sh
 
https://github.com/ultralytics/yolov5/blob/ba18528b4737a4b08b55653c54f3d3e830f8e151/data/scripts/get_voc.sh#L1-L139
